docker-compose.yml

```diff
@@ -24,6 +28,8 @@ services:
     image: postgres
     ports:
       - 5432:5432
+    volumes:
+      - postgres-data:/var/lib/postgresql/data
```

Comment:
I would use `.medtagger/postgres-data` and please add `.medtagger` to .gitignore.